tests/test_image_redis_cache.py

```diff
@@ -84,3 +84,24 @@ def test_cache_flush(self):
         self.cache.flush()
         for i in [1, 2, 3]:
             self.assertEqual(self.cache.get('foo_{0}'.format(i)), None)
+
+    def test_default_prefix_for_redis(self):
+        """Test default redis prefix"""
+        # Test default prefix for redis keys (when nothing set in config)
+        self.assertEqual(self.cache.cache.key_prefix, 'iiif')
+
+    def test_redis_prefix_set_properly(self):
+        """Test if ImageRedisCache properly sets redis prefix"""
+        from flask import current_app
+        from flask_iiif.cache.redis import ImageRedisCache
+
+        # Store old prefix
+        old_prefix = current_app.config.get('IIIF_CACHE_REDIS_PREIFX')
+        # Set new prefix in config
+        current_app.config['IIIF_CACHE_REDIS_PREIFX'] = "TEST_PREFIX"
+        # Create new ImageRedisCache which should read new prefix from config
+        tmp_redis_cache = ImageRedisCache()
+        # Check prefix set in ImageRedisCache object
+        self.assertEqual(tmp_redis_cache.cache.key_prefix, 'TEST_PREFIX')
+        # Restore old prefix in config
+        current_app.config['IIIF_CACHE_REDIS_PREIFX'] = old_prefix
```

Comment:
What do you think about writing a test which does exactly [what Chiara describes in the issue](https://github.com/inveniosoftware/flask-iiif/issues/55#issue-406827827)? ☺️
> delete all of the keys generated by iiif by simply running the flush method

Reply:
:+1: